pandasai/helpers/data_sampler.py

```diff
@@ -10,7 +10,8 @@
 import random
 
 import numpy as np
-import pandas as pd
+
+import pandasai.pandas as pd
```

Comment:
So if I understand correctly, to make it work with modin we need to import pandas like this. While if we import with `import pandas as pd` it will still work, but just ignore the logic to optionally import it with modin, right?

Reply:
@mspronesti while if one, for example, `import pandas as pd`, it will still work, right? What was the issue with `import modin.pandas as pd`?

Reply:
@gventuri From the user perspective, pandasai is still supposed to be used with pandas or modin, i.e. one would just import one of the two in their snippet and use pandasai as they were before. 

However, internally this import is needed because `pandasai.pandas` contains the logic to select the engine, which is triggered whenenver `pandasai.set_pd_engine("modin|pandas")` is called.

Reply:
@mspronesti thanks a lot for clarifying, makes a lot of sense! I'll have a final review later today and merge it! Thanks a lot for the great improvement, super helpful for larger datasets!